client/my-sites/post-type-filter/index.jsx

```diff
@@ -241,6 +248,7 @@ export default flow(
 
 			return {
 				...props,
+				typeLabel: get( getPostType( state, siteId, query.type ), 'labels.name' ),
```

Comment:
Should we return a default here? Or at least, handle the case where `labels.name` may not exist?

Reply:
If `labels.name` does not exist, the placeholder will be `Search ...` (notice the whitespace), which is OK IMHO.

Reply:
sounds good!

Reply:
Yeah, not 100% ideal, but good enough I think.